### ISSUE TITLE
2 DMX outputs

### ISSUE THREAD

**Issue by ghost** [Aircoookie/WLED#1893]
Hi! Is it possible to output two dmx universes on wled??
I think that in order to get it the current dmx library has to be changed to other that supports 2 outputs.
Any thoughts?
Thanks.

**Comment by huggy-d1** [comment]
If your wled instance has more than 510 LEDs configured, I believe the 2nd universe starts at LED #511. You can search the https://wled.discourse.group/ site or this issues (both open and closed) for "DMX universe".

**Comment by Ekim16** [comment]
Yes, I know that. I mean DMX when modifying the code with `#define WLED_ENABLE_DMX` not the pixels.

**Comment by push-pop** [comment]
Hello! This should be possible and I am looking into it now...

I am using this board:
https://www.tindie.com/products/inpublic/esp8266nodemcu-2-universe-dmx-board/

With ESP8266.

I plan to

1. Make DMX pin configurable
2. Add a second DMX channel.

I'll be putting my work here: 
https://github.com/push-pop/WLED/tree/feature/dmx_multi_universe
